src/EditorFeatures/Core/CommandHandlers/QuickInfoCommandHandlerAndSourceProvider.cs

```diff
@@ -35,17 +35,17 @@ internal partial class QuickInfoCommandHandlerAndSourceProvider :
         public QuickInfoCommandHandlerAndSourceProvider(
             [ImportMany] IEnumerable<Lazy<IQuickInfoProvider, OrderableLanguageMetadata>> providers,
             [ImportMany] IEnumerable<Lazy<IAsynchronousOperationListener, FeatureMetadata>> asyncListeners,
-            [ImportMany] IEnumerable<Lazy<IIntelliSensePresenter<IQuickInfoPresenterSession, IQuickInfoSession>, OrderableMetadata>> presenters)
-            : this(ExtensionOrderer.Order(presenters).Select(lazy => lazy.Value).FirstOrDefault(),
+            [ImportMany] IEnumerable<Lazy<IIntelliSensePresenter<IQuickInfoPresenterSession>, OrderableMetadata>> presenters)
+            : this(ExtensionOrderer.Order(presenters).Select(lazy => lazy.Value).OfType<IIntelliSensePresenter<IQuickInfoPresenterSession, IQuickInfoSession>>().FirstOrDefault(),
```

Comment:
I removed the dependencies on `ICompletionSession` and `ISignatureHelpSession`. I did not remove the dependency on `IQuickInfoSession`.